### PR TITLE
TRITON-2276 sdc-headnode USBs need to be 8GB now

### PR DESCRIPTION
**Pull request by danmcd**
Cannot go back until OS-8363 does.

### CODE REVIEW

File: bin/build-coal-image [hunk 1]
```diff
@@ -73,8 +73,8 @@ exec > >(tee "${LOGFILE}") 2>&1
 . "${ROOT}/buildtools/lib/trace_logger.sh"
 
 BUILD_TGZ=$(build_spec build-tgz)
-USB_SIZE="4gb"
-USB_SKIP=7812500
+USB_SIZE="8gb"
+USB_SKIP=15625000
```
Comment (bahamat) [inline bin/build-coal-image:77]: This is in bytes, so we're upping it from 7.8MB to 15.6MB. That's probably an unnecessary change, but I also don't think it hurts anything.

Comment (danmcd) [reply bin/build-coal-image:77]: I'd rather be safe than sorry here, but I can rebuild with it set to 7812500 quickly enough to see.

Comment (danmcd) [reply bin/build-coal-image]: Reverted.

Comment (bahamat) [reply bin/build-coal-image]: Tested output image, and we're good there.

**Review by bahamat — APPROVED** [review]
